cibuildwheel/macos.py

```diff
@@ -47,31 +47,41 @@ def call(args, env=None, cwd=None, shell=False):
 
         # if this version of python isn't installed, get it from python.org and install
         python_package_identifier = 'org.python.Python.PythonFramework-%s' % config.version
+        installation_bin_path = '/Library/Frameworks/Python.framework/Versions/{}/bin'.format(config.version)
```

Comment:
This can now be moved down a little bit (as I'd first done, before moving the symlinks into the if statement of the installation), to just before the `if config.version[0] == '3'`. It feels slightly more relevant and in place there.